BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -237,10 +237,12 @@ - (void)didReceiveAccessTokenRefreshedNotification:(NSNotification *)notificatio
 
 - (void)didReceiveUserWasLoggedOutNotification:(NSNotification *)notification
 {
-    NSDictionary *userInfo = (NSDictionary *)notification.object;
-    NSString *userID = [userInfo objectForKey:BOXUserIDKey];
-    if ([userID isEqualToString:self.user.modelID]) {
-        [self logOut];
+    if ([notification.object isKindOfClass:[NSDictionary class]]) {
```

Comment:
I think this was caused by commit dda2df7eff30 (part of the App Users work).

Can we just do a refactor and stop using the 'object' for this. It really should go into userInfo. I believe there's only one component that posts this notification in BOXAPIDataOperation. It'd be great if we could just clean it up.